packages/web3-providers-ws/test/unit/supportsSubscriptions.test.ts

```diff
@@ -2,7 +2,7 @@ import Web3ProviderWS from '../../src/index';
 import { WebSocketOptions } from '../../src/types';
 
 describe('web3ProvidersWS.supportsSubscriptions', () => {
-    it('should return true', () => {
+    it('should return true', async () => {
```

Comment:
I don't think you any `async` here, and the mocking of `connect` needs to be done here as well

Reply:
added comments for where to use mock above